topics/icrc_29_window_post_message_transport.md

```diff
@@ -0,0 +1,57 @@
+# ICRC-29: Window Post Message Transport for ICRC-25
+
+| Status |
+|:------:|
+| Draft  |
+
+## Summary
+
+This standard defines a transport channel to send ICRC-25 messages from a relying party to a signer. The transport channel is based on the [window.postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) API.
+
+## Terminology
+
+* signer: A service that manages a user's keys and can sign and perform canister calls on their behalf.
+* relying party: A service that wants to request calls on a specific canister.
+
+## Trust Assumption
+For this standard to represent an [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md) compliant transport channel, the following assumptions must hold:
+* The users machine is not compromised. In particular, the browser must deliver messages unchanged and represent the `origin` of the different parties correctly.
+* DNS entries are not compromised. The users machine must be able to resolve the domain names of the relying party and signer correctly.
+
+## Establishing the Communication Channel
```

Comment:
I'd say "Establishing a Communication Channel" (and start the paragraph below with "A communication channel"), but maybe ask a native speaker :-)